clang-tools-extra/clangd/CodeComplete.cpp

```diff
@@ -89,7 +89,9 @@ const CodeCompleteOptions::CodeCompletionRankingModel
 
 namespace {
 
-CompletionItemKind toCompletionItemKind(index::SymbolKind Kind) {
+CompletionItemKind
+toCompletionItemKind(index::SymbolKind Kind,
+                     const llvm::StringRef *Signature = nullptr) {
```

Comment:
nit: can you add a comment stating that this function needs to remain synchronized with the one below?

Reply:
Done, thanks